### PR TITLE
Update Swiss Hydrological Data documentation

### PR DESCRIPTION
**Pull request by Bouni**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17166

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.swiss_hydrological_data.markdown [hunk 1]
```diff
@@ -13,15 +13,15 @@ ha_iot_class: "Cloud Polling"
 ha_release: 0.22
 ---
 
-<p class='note warning'>
-  This sensor doesn't work at the moment due to changed by the [Swiss Federal Office for the Environment (Bundesamt für Umwelt - Abt. Hydrologie)](http://www.hydrodaten.admin.ch) to access the data.
+<p class='note info'>
```
Comment (WofWca) [inline source/_components/sensor.swiss_hydrological_data.markdown]: IIRC `class='note'` is enough.

File: source/_components/sensor.swiss_hydrological_data.markdown [hunk 2]
```diff
@@ -44,24 +46,46 @@ name:
...
+  - The maximum discharge measurement of the last 24 hours
+- mean_temperature
+  - The mean temperature measurement of the last 24 hours
+- mean_level
+  - The mean water level measurement of the last 24 hours
+- mean_discharge
+  - The mean discharge measurement of the last 24 hours
 
-This sensor contains additional information which an easily accessed by a [template sensor](/components/sensor.template/).
+<p class='note info'>
```
Comment (WofWca) [inline source/_components/sensor.swiss_hydrological_data.markdown:61]: Same

File: source/_components/sensor.swiss_hydrological_data.markdown [hunk 3]
```diff
@@ -44,24 +46,46 @@ name:
   required: false
   type: string
   default: 
+measurements:
+  description: list of measurements you want to use.
+  required: true
+  type: list
+  deafult:
```
Comment (frenck) [inline source/_components/sensor.swiss_hydrological_data.markdown]: Typo: `default`. Also, default should have a value.

File: source/_components/sensor.swiss_hydrological_data.markdown [hunk 4]
```diff
@@ -44,24 +46,46 @@ name:
...
 {% endconfiguration %}
```
Comment (frenck) [inline source/_components/sensor.swiss_hydrological_data.markdown:47]: This is a list and that should have a `keys` attribute. You can use that attribute to list the possible measurement values.

File: source/_components/sensor.swiss_hydrological_data.markdown [hunk 5]
```diff
@@ -44,24 +46,46 @@ name:
...
 
-The hydrological measurings are coming from the [Swiss Federal Office for the Environment (Bundesamt für Umwelt - Abt. Hydrologie)](http://www.hydrodaten.admin.ch) and are updated almost in real-time.
+Valid measurement values are:
 
-## {% linkable_title Example %}
+- temperature
```
Comment (frenck) [inline source/_components/sensor.swiss_hydrological_data.markdown]: Move this section into the `keys` second of the `measurements` configuration value.

File: source/_components/sensor.swiss_hydrological_data.markdown [hunk 6]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "Swiss Hydrological Data"
 description: "Instructions on how to integrate hydrological data of Swiss waters within Home Assistant."
-date: 2016-06-17 17:00
+date: 2018-10-05 12:00
```
Comment (frenck) [inline source/_components/sensor.swiss_hydrological_data.markdown]: Please do not update the date. It marks the initial introduction.

**Review by frenck — DISMISSED** [review]
@Bouni Thank you for this PR. Some comments were made on it. Could you please take a look? 👍

**Comment by Bouni** [comment]
I've got some feedback on the component itself. Therefore I will update fix the documentation as soon as I've found a solution for the addressed issues with the component!

**Comment by frenck** [comment]
Thanks for letting us know @Bouni!
We'll wait patiently 👍

**Comment by frenck** [comment]
Hold on sec, this PR depends on a parent PR. 
This docs PR should go into the `next` branch.
@Bouni Please be sure to update the target and rebase your changes.

**Comment by Bouni** [comment]
@frenck I've used **current** because the documentation says that updates of existing components should go into **current**, just documentation on new components should go into **next**.
Anyway, I'll do what you said :+1:

**Comment by frenck** [comment]
@Bouni I understand the confusion.

Updates to the current release or documentation fixes go into the `current` branch. Changes that depend on a next release go into `next`. Since you are changing code (which results in changing docs), the change should go into `next`. That documentation will go live when the next release takes place.

This ensures the docs & release are in sync (and matches).

**Review by fabaff — APPROVED** [review]
:bird:

**Pull request merged**